xcode/Subconscious/Shared/Components/Detail/MemoViewerDetailView.swift

```diff
@@ -561,10 +561,11 @@ struct MemoViewerDetailModel: ModelProtocol {
                         .profile
                 }
                 
+                let did = try await environment.noosphere.resolve(peer: state.address?.peer)
+                
                 return try await environment
                     .userProfile
-                    .requestUserProfile(petname: petname)
```

Comment:
The viewer was hitting the author's profile memo just to load optional metadata, I've changed it to try and read from the DB.